### ISSUE TITLE
test: Missing argument at index 2 error

### ISSUE THREAD

**Issue by mrThe** [oh-my-fish/plugin-rbenv#6]
I've just updated omf (`omf self-update`), update all plugins (`omf update`) and then get this:

I've get `test: Missing argument at index 2` when open new tab on terminal. It appears only if `rbenv` plugin is turned on.

`OS X 10.9.5;
fish, version 2.2.0`

**Comment by bpinto** [comment]
I couldn't find any call to test on this plugin. May it be your theme?

**Comment by mrThe** [comment]
@bpinto no, it's not theme.

Part of fish config:

``` bash
# Load oh-my-fish configuration.
. $fish_path/oh-my-fish.fish

# Theme
# Theme "robbyrussell"

# Plugins
Plugin "brew"
Plugin "bundler"
Plugin "gem"
Plugin "osx"
## Plugin "rails"
# Broken:
Plugin "rbenv"
```

``` bash
test: Missing argument at index 2
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
mrthe@mrMac ~>
```

But, when i do that:

``` bash
# Load oh-my-fish configuration.
. $fish_path/oh-my-fish.fish

# Theme
Theme "robbyrussell"

# Plugins
Plugin "brew"
Plugin "bundler"
Plugin "gem"
Plugin "osx"
## Plugin "rails"
# Broken:
# Plugin "rbenv"
```

Everything is fine.

How can i debug it, or see more details about this error?

**Comment by bpinto** [comment]
It seems like you are using an old version of omf, could you see if upgrading fixes it?

**Comment by mrThe** [comment]
So, my steps:

1) Remove rbenv folder `rm -rf ~/.oh-my-fish/plugins/rbenv`
2) Open new terminal tab, gets `Plugin 'rbenv' is not installed. Run 'omf install' to download and install it.`
3)

``` bash
➜ omf self-update
Username for 'https://github.com': Oh My Fish has been successfully updated.
➜ omf install
Installing rbenv... ✔
```

4) Open new terminal tab, still gets `test: Missing argument at index 2`

**Comment by mrThe** [comment]
Ah, damn it. 

``` bash
Username for 'https://github.com': Oh My Fish has been successfully updated.
```

I have an old repo url in my `~/.oh-my-fish/.git/config`. So, looks like omf never updates for me. And now i have some other issues related to omf, but this issue will be closed for now.

**Comment by mrThe** [comment]
I've updated omf to latest version ( https://github.com/oh-my-fish/oh-my-fish/issues/33 ) . And now, i get this:

``` bash
➜ omf install rbenv
Installing rbenv package...
✔ package rbenv successfully installed.
function rbenv
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
➜ 
```

Then, open new tab and get this:

``` bash
function rbenv
Welcome to fish, the friendly interactive shell
Type help for instructions on how to use fish
➜
```

`function rbenv` - what is that?

ps. i reopen issue, but i can create another one if it necessary.

**Comment by bobthecow** [comment]
It's #7 (fix is in #10)

**Comment by mrThe** [comment]
Thanks. Closed.
